tests/pull.bats

```diff
@@ -27,102 +27,103 @@ load helpers
+  run buildah pull --registries-conf ${TESTSDIR}/registries.conf --signature-policy ${TESTSDIR}/policy.json busybox
   echo "$output"
   [ "$status" -eq 0 ]
   run buildah images --format "{{.Name}}:{{.Tag}}"
   echo "$output"
+  [ "$status" -eq 0 ]
   [[ "$output" =~ "busybox:glibc" ]]
   [[ "$output" =~ "busybox:latest" ]]
-  run buildah pull --signature-policy ${TESTSDIR}/policy.json quay.io/libpod/alpine_nginx:latest
+  run buildah pull --registries-conf ${TESTSDIR}/registries.conf --signature-policy ${TESTSDIR}/policy.json quay.io/libpod/alpine_nginx:latest
   echo "$output"
   [ "$status" -eq 0 ]
   run buildah images --format "{{.Name}}:{{.Tag}}"
   echo "$output"
+  [ "$status" -eq 0 ]
   [[ "$output" =~ "alpine_nginx:latest" ]]
   run buildah rmi quay.io/libpod/alpine_nginx:latest
   echo "$output"
   [ "$status" -eq 0 ]
-  run buildah pull --signature-policy ${TESTSDIR}/policy.json quay.io/libpod/alpine_nginx
+  run buildah pull --registries-conf ${TESTSDIR}/registries.conf --signature-policy ${TESTSDIR}/policy.json quay.io/libpod/alpine_nginx
   echo "$output"
   [ "$status" -eq 0 ]
   run buildah images --format "{{.Name}}:{{.Tag}}"
   echo "$output"
+  [ "$status" -eq 0 ]
   [[ "$output" =~ "alpine_nginx:latest" ]]
-  run buildah pull --signature-policy ${TESTSDIR}/policy.json alpine@sha256:1072e499f3f655a032e88542330cf75b02e7bdf673278f701d7ba61629ee3ebe
+  run buildah pull --registries-conf ${TESTSDIR}/registries.conf --signature-policy ${TESTSDIR}/policy.json alpine@sha256:1072e499f3f655a032e88542330cf75b02e7bdf673278f701d7ba61629ee3ebe
   echo "$output"
   [ "$status" -eq 0 ]
-  run buildah pull --signature-policy ${TESTSDIR}/policy.json fakeimage/fortest
+  run buildah pull --registries-conf ${TESTSDIR}/registries.conf --signature-policy ${TESTSDIR}/policy.json fakeimage/fortest
   echo "$output"
   [ "$status" -ne 0 ]
   run buildah images --format "{{.Name}}:{{.Tag}}"
   echo "$output"
+  [ "$status" -eq 0 ]
   [[ ! "$output" =~ "fakeimage/fortest" ]]
 }
 
 @test "pull-from-docker-archive" {
   run buildah pull --signature-policy ${TESTSDIR}/policy.json alpine
   echo "$output"
   [ "$status" -eq 0 ]
-  run buildah push --signature-policy ${TESTSDIR}/policy.json docker.io/library/alpine:latest docker-archive:/tmp/alp.tar:alpine:latest
+  run buildah push --signature-policy ${TESTSDIR}/policy.json docker.io/library/alpine:latest docker-archive:${TESTDIR}/alp.tar:alpine:latest
   echo "$output"
   [ "$status" -eq 0 ]
   run buildah rmi alpine
   echo "$output"
   [ "$status" -eq 0 ]
-  run buildah pull --signature-policy ${TESTSDIR}/policy.json docker-archive:/tmp/alp.tar
+  run buildah pull --signature-policy ${TESTSDIR}/policy.json docker-archive:${TESTDIR}/alp.tar
   echo "$output"
   [ "$status" -eq 0 ]
   run buildah images --format "{{.Name}}:{{.Tag}}"
   echo "$output"
-  [[ "$output" =~ "alpine" ]]
-  run rm -f /tmp/alp.tar
-  echo "$output"
   [ "$status" -eq 0 ]
+  [[ "$output" =~ "alpine" ]]
```

Comment:
does ${TESTDIR}/alp.tar get auto removed?  I think so, but just want to double check.

Reply:
on teardown TESTDIR is rm completely.

Reply:
Thanks for the confirmation @giuseppe , I thought that was the case but couldn't remember last night.